droid-command-line/src/main/java/uk/gov/nationalarchives/droid/command/action/CommandLineParam.java

```diff
@@ -63,6 +63,14 @@ public DroidCommand getCommand(CommandFactory commandFactory, CommandLine cli) {
         }
     },
 
+    /** Clean. */
+    CLEAN("C", "clean", I18N.CLEAN_HELP) {
```

Comment:
1. Maybe `CLEAN_HOME_DIR("C", "clean-home", I18N.CLEAN_HELP)`?
2. Why not a lowercase `c`?

Reply:
1. The command is called "clean" which is the second argument and the enum entry CLEAN matches it.
2. It's already used in this file.